docs/api/connectivity/bluetooth/Optimising_for_performance.md

```diff
@@ -0,0 +1,173 @@
+# Optimising application for throughput and power consumption
+
+BLE devices are usually battery powered so performance might mean different things in different applications. You have
```

Comment:
I know it was spelled out in an earlier chapter, but consider spelling out Bluetooth Low Energy (BLE) in the first use here...